### PR TITLE
Create Stock Statement page 

### PR DESCRIPTION
**Pull request by AizaAnjum**
Kindly create new page of Stock Statement as per wireframe 

https://app.diagrams.net/#Wb!HT-wfrbGcEqM5DoBIQXmPY19VMxuO41MpyWO-AFCZ9OE0Sc7-s_LQpcniXtoUBo4%2F01ZZKZDJSFHBFVK66RZVHIRLBZ73QZA453

### CODE REVIEW

**Comment by AizaAnjum** [comment]
Screenshot 2021-10-20 at 2 08 44 AM

File: src/components/analytics/Basic.vue [hunk 1]
```diff
@@ -95,6 +102,9 @@ export default defineComponent({
         start_date: this.startDate,
         end_date: this.endDate,
       });
+    },
+    redirectToStockStatement() {
+      this.$router.push({path: '/analytics/stock'});
```
Comment (OmerAli277) [inline src/components/analytics/Basic.vue]: Any reason for this function?

**Review by OmerAli277 — CHANGES_REQUESTED** [review]
Make the layout more consistent with the wireframe.
Read about Grid CSS and check how it is used on the order page.
Apply grid CSS on the top selection bar for a consistent design.

File: src/components/analytics/Basic.vue (hunk 1 above)
Comment (AizaAnjum) [reply src/components/analytics/Basic.vue]: I initially created a button to route to stock statement page and this function was linked to it but then i integrated stock statement with the tabs. so i will remove the function now

**Comment by AizaAnjum** [comment]
Changes made @OmerAli277

**Review by OmerAli277 — APPROVED** [review]

**Pull request merged**